src/app/components/AccountInfo/NodeUriModal.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Modal } from 'antd';
+import { getNodeInfo } from 'modules/node/actions';
+import QRCode from 'qrcode.react';
+import Copy from 'components/Copy';
+import { AppState } from 'store/reducers';
+import './DepositModal.less';
+
+interface StateProps {
+  node: AppState['node']['nodeInfo'];
+}
+
+interface DispatchProps {
+    getNodeInfo: typeof getNodeInfo;
+  }
+
+interface OwnProps {
+  isOpen?: boolean;
+  onClose(): void;
+}
+
+type Props = StateProps & OwnProps;
+
+class NodeUriModal extends React.Component<Props> {
+
+  render() {
+    const {
+      node,
+      isOpen,
+      onClose
+    } = this.props;
+    const text = node!.uris[0] === undefined ? node!.identity_pubkey : node!.uris[0];
```

Comment:
`node` could be undefined here, that should probably be handled instead of ignored, even if it's just a `return null`. You also probably want to do a falsy check here instead of undefined, in case it got set to null.

Reply:
```Typescript
const text = node === null ? '' : !!node.uris[0] === false ? 
  node.identity_pubkey : node.uris[0];
const title = node === null ? '' : !!node.uris[0] === false ? 
  'Pubkey' : 'Node URI';